test/routes/article-list.test.ts

```diff
@@ -3,23 +3,65 @@ import { Next, Response } from 'koa';
 import articleList from '../../src/routes/article-list';
 import createContext from '../context';
 import runMiddleware from '../middleware';
+import InMemoryArticles from '../../src/adaptors/in-memory-articles';
 
-const makeRequest = async (next?: Next): Promise<Response> => {
+const makeRequestEmptyLst = async (next?: Next): Promise<Response> => {
   const context = createContext();
+  const articles = new InMemoryArticles();
 
-  return runMiddleware(articleList(context.router), context, next);
+  return runMiddleware(articleList(articles, context.router), context, next);
+};
+
+const makeRequestList = async (next?: Next): Promise<Response> => {
+  const context = createContext();
+  const articles = new InMemoryArticles();
+
+  (async (): Promise<void> => {
+    await Promise.all([
+      articles.add({
+        '@type': 'http://schema.org/Article',
+        '@id': '/articles/09560',
```

Comment:
Since these aren't dereferenceable yet, we should use a [blank node](https://www.w3.org/TR/rdf11-concepts/#h3_section-blank-nodes), so like `'@id': '_:12345'`.

To make the Hydra console work, we need to let Hydra know about the class too. So the API documentation will need:

```typescript
{
  '@id': schema.Article,
  '@type': hydra.Class,
  [hydra.title]: { '@value': 'Article', '@language': 'en' },
  [hydra.supportedProperty]: [
    {
      '@type': hydra.SupportedProperty,
      [hydra.title]: { '@value': 'Name', '@language': 'en' },
      [hydra.property]: {
        '@id': schema.name,
        '@type': rdf.Property,
      },
      [hydra.required]: true,
      [hydra.readable]: true,
      [hydra.writeable]: true,
    },
  ],
},
```

(Try then adding an article manually in `app.ts` and see it appear.)